src/cpr_data_access/vespa.py

```diff
@@ -0,0 +1,201 @@
+from pathlib import Path
+from typing import List
+
+import yaml
+from vespa.io import VespaResponse
+
+from cpr_data_access.models.search import (
+    Family,
+    Hit,
+    SearchRequestBody,
+    SearchResponse,
+    filter_fields,
+    sort_fields,
+)
+
+
+def _split_document_id(document_id: str) -> tuple[str, str, str]:
```

Comment:
Is there any reason half these functions are suggested as private?

Reply:
I think in a previous version of the code, a lot of these _were_ intended as private methods on some class. Happy to get rid of those leading underscores now though 😁